docs/guides/getting-started-java-spring.md

```diff
@@ -35,16 +31,26 @@ For example, in this guide we will outline a BPMN model to receive a payment req
 While stepping through this guide, you can visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code to check your work.
 :::
 
-## Step 1: Install Camunda 8 Self-Managed
+## Prerequisites
+
+Before getting started, ensure you have the following in your local environment:
 
-<Install/>
+- Access to your preferred code editor or IDE
+- [OpenJDK 21+](https://openjdk.org/install/)
+- Camunda 8 Desktop Modeler and Camunda 8 Run
```

Comment:
Can we link to the docs for both here?

Reply:
I will incorporate this in a separate PR @conceptualshark 👍